views/record_timestamp.go

```diff
@@ -0,0 +1,30 @@
+package views
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ugent-library/biblio-backoffice/ctx"
+	"github.com/ugent-library/biblio-backoffice/models"
+)
+
+func CreatedBy(c *ctx.Ctx, createdAt *time.Time, createdBy *models.Person) string {
+	s := fmt.Sprintf("Created %s", createdAt.In(c.Timezone).Format("2006-01-02 15:04"))
+	if createdBy != nil {
+		s += " by " + createdBy.FullName
+	}
+	s += "."
+	return s
```

Comment:
@nicolasfranck Maybe you can use [`lo.Ternary`](https://pkg.go.dev/github.com/samber/lo#readme-ternary) here?